src/config/chis-tg/config.json

```diff
@@ -0,0 +1,196 @@
+          "property_name": "first_name",
+          "type": "string",
+          "parameter": ["\\(Chw First Name\\)", "\\(Chw Firstname\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "Full Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "{{ contact.last_name }} {{ contact.first_name }}",
+          "required": true
+        },
+        {
+          "friendly_name": "Gender",
+          "property_name": "sex",
+          "type": "gender",
+          "required": true
+        },
+        {
+          "friendly_name": "Telephone",
+          "property_name": "telephone",
+          "type": "phone",
+          "parameter": "TG",
+          "required": true
+        },
+        {
+          "friendly_name": "Appartenance NGO/Structure",
+          "property_name": "appartenance_ngo",
+          "type": "string",
+          "parameter": ["\\(STRUCTURE NAME\\)", "\\(STRUCTURE-NAME\\)"],
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "ci90_chw_site",
+      "friendly": "CHW Post",
+      "contact_type": "ci92_chw_site",
+      "user_role": ["chw", "community_user"],
+      "username_from_place": true,
+      "deactivate_users_on_replace": true,
+      "hierarchy": [
+        {
+          "friendly_name": "Village",
+          "property_name": "village",
+          "contact_type": "ch80_village",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected CHW Post",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(Chw Post Name\\)", "\\(CHW POST NAME\\)", "\\(POST NAME\\)", "\\(Post Name\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "CHW Post Name",
+          "property_name": "name",
+          "type": "string",
+          "parameter": ["\\(Chw Post Name\\)", "\\(CHW POST NAME\\)", "\\(POST NAME\\)", "\\(Post Name\\)"],
+          "required": true
+        }
+      ],
+      "contact_properties": [
+        {
+          "friendly_name": "CHW Last Name",
+          "property_name": "last_name",
+          "type": "string",
+          "parameter": ["\\(CHW LAST NAME\\)", "\\(CHW LASTNAME\\)", "\\(CHW-LASTNAME\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "CHW First Name",
+          "property_name": "first_name",
+          "type": "string",
+          "parameter": ["\\(Chw First Name\\)", "\\(Chw Firstname\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "CHW Full Name",
+          "property_name": "name",
+          "type": "string",
+          "parameter": "{{ contact.last_name }} {{ contact.first_name }}",
+          "required": true
+        },
+        {
+          "friendly_name": "Agent Type",
+          "property_name": "role",
```

Comment:
should this be removed now since it is handled by `user_role`?

Reply:
I have recently noticed this in all the `moh-togo-config` project, but we will handle at the other side. there `user_role` is copied in the contact and used in many part of the config. 
I have planned to refactor this and have an [issue](https://github.com/medic/config-moh-togo/issues/253) to track that.